src/lib/common/JsonHelper.cpp

```diff
@@ -149,6 +154,46 @@ void JsonHelper::addRaw(const std::string& key, const std::string& value)
 }
 
 
+/* ****************************************************************************
+*
+* JsonHelper::addDouble -
+*/
+void JsonHelper::addLongLong(const std::string& key, long long value)
+{
+  if (!empty)
+  {
+    ss << ',';
+  }
+  ss << toJsonString(key) << ':' << value;
+
+  empty = false;
+}
+
+
+/* ****************************************************************************
+*
+* JsonHelper::addDate -
+*/
+void JsonHelper::addDate(const std::string& key, long long timestamp)
+{
+  if (!empty)
+  {
+    ss << ',';
+  }
+
+  // 80 bytes are large enough to store any ISO8601 string safely
```

Comment:
"1970-04-26T17:46:40.00Z" is 24 char long. Event assuming some more space for the timezone in the future, I think that 80 bytes will suffice al cases.